Leanplum-SDK/Classes/Managers/Networking/LPAPIConfig.h

```diff
@@ -0,0 +1,27 @@
+//
+//  LPAPIConfig.h
+//  Leanplum-iOS-SDK-source
```

Comment:
I think "Config" is might be ambiguous here. Could we get a description in here to clarify what is being stored? Is it `LPAPIArguments`?